### ISSUE TITLE
[Feature request] View local database

### ISSUE THREAD

**Issue by pescepalla** [bwt/OpenPaperView#5]
Thanks for this. I use openpaper.work on my workstations and have been looking for a mobile client since forever.

Are you planning to introduce a local database feature?

I don't have a server but I sync my documents with Syncthing and it would be great if OpenPaperView could access the database in this way.

**Comment by bwt** [comment]
Indeed, it would be nice to have an option that does not require a server.
I will look into it.

**Comment by lrnzo** [comment]
i would be interested in this feature too. i sync everything with nextcloud
